api/all/src/main/java/io/opentelemetry/api/trace/Span.java

```diff
@@ -162,6 +162,23 @@ default Span setAttribute(AttributeKey<Long> key, int value) {
     return setAttribute(key, (long) value);
   }
 
+  /**
+   * Sets attributes to the {@code Span}. If the {@code Span} previously contained a mapping for any
+   * of the keys, the old values are replaced by the specified values.
+   *
+   * @param attributes the attributes
+   * @return this.
+   */
+  @SuppressWarnings("unchecked")
+  default Span setAllAttributes(Attributes attributes) {
+    if (attributes == null || attributes.isEmpty()) {
+      return this;
+    }
+    attributes.forEach(
+        (attributeKey, value) -> this.setAttribute((AttributeKey<Object>) attributeKey, value));
```

Comment:
`s/Object/?`?

Reply:
`attributeKey` is already an `AttributeKey<?>`, it needs to be unchecked cast to `AttributeKey<Object>` in order to call `setAttribute` with `value` since it is an `Object`.